pallets/Polkadex_Ido/src/lib.rs

```diff
@@ -625,4 +645,25 @@ impl<T: Config> Module<T> {
     pub fn pallet_account_id() -> T::AccountId {
         T::ModuleId::get().into_account()
     }
-}
+
+    pub fn rounds_by_investor(account : T::AccountId) -> Vec<(T::Hash, FundingRoundWithPrimitives)> {
+        <InvestorShareInfo<T>>::iter().filter_map(|(round_id, investor, _)| {
+            if investor != account {
```

Comment:
This will be a linear search over all rounds and all investors, How will this scale up?